.gitignore

```diff
@@ -258,3 +258,7 @@ saas_config.toml
 # Frontend files
 **/node_modules
 **/.next
+
+
+# multilang migration backup files produced by db init
+multilang_migration_backup_*
```

Comment:
not related to this change directly, just added this because i'd noticed these building up on my local and it was getting messy!